notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>I would say:</p><p>because the world of manifolds is completely different than the world of Euclidean space</p><p>or:</p><p>because a manifold is completely different from a Euclidean spacee</p><p>instead of:</p><p>because the space of manifolds is completely different than the space of Euclidean space</p><p>^^^ "the space of manifolds" is a bit meta! (the manifold of manifolds??! :p )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>hahaha, true! </p>